mathext/ell_carlson.go

```diff
@@ -0,0 +1,145 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"math"
+)
+
+// EllipticRF computes the symmetric elliptic integral R_F(x,y,z):
+//
+//	R_F(x,y,z) = (1/2)\int_{0}^{\infty}{1/s(t)} dt,
+//	s(t) = \sqrt{(t+x)(t+y)(t+z)}.
+//
+// See: http://dlmf.nist.gov/19.16.E1 for the definition.
+//
+// See: http://doi.org/10.1145/355958.355970 for the original Fortran code.
+//
+// See: http://dx.doi.org/10.1007/BF02198293 for the modified method of computation.
```

Comment:
There is also https://arxiv.org/abs/math/9409227
Is its contents the same? It would be better to link to something that is not behind a paywall.